bus-board/src/components/BusDataTable.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {IJsonBus} from "../modules/IJsonBus";
+import {BusDataRow} from "./BusDataRow";
+
+export const BusDataTable: React.FC<{ bussesData: IJsonBus[], buttonPressed: boolean }> = ({bussesData, buttonPressed}) => {
+
+    const rows: React.ReactElement[] = [];//scapam de any mai tarziu

```

Comment:
Comment

Reply:
I think we got rid of the any type before last commit, deleted comment